templates/bareos-sd.conf.j2

```diff
@@ -12,4 +12,7 @@ Storage {
 {% if bareos_sd_tls_verify_peer %}
   TLS Verify Peer = Yes
 {% endif %}
+{% if bareos_sd_device_reserve_by_media_type %}
```

Comment:
This variable now returns `undefined`, so either:

- Set it in `defaults/main.yml`
- Test if it's defined and true, something like: `{% if bareos_sd_device_reserve_by_media_type and bareos_sd_device_reserve_by_media_type %}`